### PR TITLE
fix(#978): Adds openshift cluster 3.7.0 and 3.7.1 in CI

### LINKED ISSUE #978: Add support to run integration tests on openshift cluster v3.7.0 and v3.7.1 in Travis.

### PR DESCRIPTION
**Pull request by dipak-pawar**
#### Changes proposed in this pull request:

- Adds oc version 3.7.0 an 3.7.1

Fixes #978

### CODE REVIEW

**Comment by bartoszmajsak** [comment]
The only one concern I have right now is that the build time will increase quite a bit. With 5 jobs (all started more or less in parallel) we were averaging around 16 minutes per build by looking at the recent trend. With 9 I'm afraid it will be +10 minutes, as I doubt we can have all 9 jobs executed in parallel. 

Can we prune some of the combinations then? WDYT @dipak-pawar @lordofthejars?

**Comment by dipak-pawar** [comment]
@bartoszmajsak, Currently we are starting same cluster for k8s and openshift category, like following
```bash
OC_VERSION="v3.6.0", CATEGORY="RequiresKubernetes"
OC_VERSION="v3.6.0" CATEGORY="RequiresOpenshift"
OC_VERSION="v3.6.1" CATEGORY="RequiresKubernetes"
OC_VERSION="v3.6.1" CATEGORY="RequiresOpenshift"
OC_VERSION="v3.7.0" CATEGORY="RequiresKubernetes"
OC_VERSION="v3.7.0" CATEGORY="RequiresOpenshift"
OC_VERSION="v3.7.1" CATEGORY="RequiresKubernetes"
OC_VERSION="v3.7.1" CATEGORY="RequiresOpenshift"
```
For now may be we can think to run k8s and openshift tests on single node if time required to finish it is more than 5 minutes with current build(15 minutes  + 5  minutes)

```bash
OC_VERSION="v3.6.0", CATEGORY="RequiresKubernetes,RequiresOpenshift"
OC_VERSION="v3.6.1" CATEGORY="RequiresKubernetes,RequiresOpenshift"
OC_VERSION="v3.7.0" CATEGORY="RequiresKubernetes,RequiresOpenshift"
OC_VERSION="v3.7.1" CATEGORY="RequiresKubernetes,RequiresOpenshift"
```

I will check what will be time required to finish if we combine two categories

File: foo.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+if [ ${OC_VERSION} == "v3.7.1" ] ; then 
+  echo $OC_VERSION
```
Comment (bartoszmajsak) [inline foo.sh]: Why do we need that? :)

File: .travis.yml [hunk 2]
```diff
@@ -43,6 +41,42 @@ before_install:
 
 install:
   - |
+      # OpenShift >= 3.7 requires nsenter https://github.com/openshift/origin/issues/17638
```
Comment (bartoszmajsak) [inline .travis.yml]: How about externalizing it to its own file?

File: foo.sh (hunk 1 above)
Comment (dipak-pawar) [reply foo.sh]: my bad, it was added mistakenly

**Pull request merged**